DSE 6.9.13 Release Notes (#222)
datastax/release-notes: DSE_6.9_Release_Notes.md

DSE 6.9.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any.
Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.

# Release notes for 6.9.13
11 August 2025

## Components versions for DSE 6.9.13
 * Apache Solr™ 6.0.1.5.2977
 * Apache Spark™ 2.5.0.6
 * Apache TinkerPop™ 3.4.15-20250603-0956e859
 * Apache Tomcat® 8.5.100
 * DSE Java Driver 1.10.0-dse-20241015 (DSE *internal-only* version)
 * Netty 4.1.119.1.dse
 * Spark JobServer 0.8.0.57

**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.

## 6.9.13 DSE Core

> * Fixed logic in the VERIFY BACKUP STORE command to remove blobs after failed read validation. (DSP-22975)

Reviewer: ```suggestion
* Fixed logic in the `VERIFY BACKUP STORE` command to remove blobs after read validation failed. (DSP-22975)
```

## 6.9.13 DSE Cassandra

> * Fixed the old ssl port disabling which might impact quorum based queries. (DSP-24871)

Reviewer: ```suggestion
* Fixed an issue that occurred when you disabled the old SSL port, which might have impacted quorum-based queries. (DSP-24871)
```

> * Fixed UCS major compaction triggered by nodetool compact to honour defined number of shards. (DSP-24886)

Reviewer: ```suggestion
* Fixed a UCS major compaction issue that occurred when you entered the `nodetool compact` command. (DSP-24886)
```

> * Fixed incremental repairs failure during compaction. (DSP-24922)

Reviewer: ```suggestion
* Fixed an issue where incremental repairs failed during compaction. (DSP-24922)
```

## 6.9.13 DSE Docker

> * Fixed docker images to correctly load client jvm properties. (DSP-24928)

Reviewer: ```suggestion
* Fixed Docker images to load client JVM properties correctly. (DSP-24928)
```
